### PR TITLE
Release fixes

### PR DESCRIPTION
**Pull request by teodanciu**
# Description

# Checklist

- [ ] Commit sequence broadly makes sense and commits have useful messages
- [ ] New tests are added if needed and existing tests are updated
- [ ] When applicable, versions are updated in `.cabal` and `CHANGELOG.md` files according to the
      [versioning process](https://github.com/intersectmbo/cardano-ledger/blob/master/RELEASING.md#versioning-process).
- [ ] The version bounds in `.cabal` files for all affected packages are updated. **If you change the bounds in a cabal file, that package itself must have a version increase.** (See [RELEASING.md](https://github.com/intersectmbo/cardano-ledger/blob/master/RELEASING.md#versioning-process))
- [ ] All visible changes are prepended to the latest section of a `CHANGELOG.md` for the affected packages. **New section is never added with the code changes.** (See [RELEASING.md](https://github.com/intersectmbo/cardano-ledger/blob/master/RELEASING.md#changelogmd))
- [x] Code is formatted with [`fourmolu`](https://github.com/fourmolu/fourmolu) (use `scripts/fourmolize.sh`)
- [x] Cabal files are formatted (use `scripts/cabal-format.sh`)
- [x] [`hie.yaml`](https://github.com/intersectmbo/cardano-ledger/blob/master/hie.yaml) has been updated (use `scripts/gen-hie.sh`)
- [ ] Self-reviewed the diff

### CODE REVIEW

File: eras/conway/impl/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ## 1.15.0.0
 
+* Change return type of `trySubmitTx` to include the fixedup transaction
```
Comment (lehins) [inline eras/conway/impl/CHANGELOG.md]: This should go into the testlib section

File: eras/alonzo/impl/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,19 @@
...
+  * replace `toPlutusScriptContext` with `toPlutusArgs`
+* `EraPlutusContext` class:
+  * remove `mkPlutusScriptContext`, `mkPlutusLanguageContext`
+  * add `mkPlutusWithContext`
+* add `toPlutusWithContext` to `Cardano.Ledger.Alonzo.Plutus.Context`
+* Move `PlutusScriptContext` type family to core
+* Add `ContextError` inhabitant to `TransactionScriptFailure`
+* Change return type of `evalTxExUnits` and `evalTxExUnitsWithLogs` to not be `Either`s
+* Add `lookupPlutusScript` and `eraLanguages` to Alonzo `Scripts`
+* Deprecate `lookupPlutusScript` in Alonzo `Evaluate`
```
Comment (lehins) [inline eras/alonzo/impl/CHANGELOG.md:17]: Thank you for adding these. Totally forgot about changelogs in CIP69 work

File: eras/conway/impl/CHANGELOG.md (hunk 1 above)
Comment (teodanciu) [reply eras/conway/impl/CHANGELOG.md]: Oh yeah! Thanks

File: eras/alonzo/impl/CHANGELOG.md (hunk 2 above)
Comment (teodanciu) [reply eras/alonzo/impl/CHANGELOG.md:17]: Happy to contribute!

**Comment by teodanciu** [comment]
Superseded by https://github.com/IntersectMBO/cardano-ledger/pull/4409

**Pull request closed without merging**